contrib/lkt/language/parser.py

```diff
@@ -129,6 +129,14 @@ class ParamMatch(Struct):
     formal = UserField(type=T.FormalParam, default_value=No(T.FormalParam))
```

Comment:
Comment about the commit itself. I guess it shouldn't be in this PR?

Reply:
I suppose you are referring to “Lkt: complete typing of "if" expressions” (27eb3ce). Yes indeed it should not be there, I’ll remove it, thank you!